docs/contributing/documentation_guidelines.md

```diff
@@ -305,7 +305,7 @@ There is some formatting in the documentation which is technically supported, bu
 
 ### Call-outs
 
-We only use `{note}` and `{warning}` call-outs. Avoid `{important}`, `{topic}`, and `{tip}`. If you find one of these, please replace it with `{note}`.
+We only use `{note}` and `{warning}` call-outs. Avoid `{admonition}`, `{important}`, `{topic}`, and `{tip}`. If you find one of these, please replace it with `{note}`.
```

Comment:
question: do we need to be explicit about not adding a title a la `{note} django-modelcluster and ParentalKey`